elements/bulbs-video/components/revealed.js

```diff
@@ -97,6 +97,10 @@ export default class Revealed extends React.Component {
       videoMeta.player_options.defaultCaptions = true;
     }
 
+    if (this.props.embedded) {
+      videoMeta.player_options.embedded = true;
+    }
+
```

Comment:
I think `this.props.embedded` is already boolean so you could just `videoMeta.player_options.embeded = this.props.embeded`, right? Is there something else I'm missing?